hw1/02-palindrome.js

```diff
@@ -1,3 +1,41 @@
-const elem = document.querySelector('input');
+// Declarations
+const elem = document.querySelector("input");
+elem.addEventListener("change", handleInput); // get input on enter
+const output = document.getElementById("result"); // output the result to HTML
 
-elem.addEventListener('input', handleInput);
+// Check for positive number
+function isPositiveNumber(num) {
+  if (num < 0) {
+    return false;
+  }
+}
+
+// Reverse a number
+function reverseNumber(num) {
+  let result = "";
+  for (i = String(num).length - 1; i >= 0; --i) {
+    result = result + num[i];
+  }
+  return result;
+}
+
+function handleInput() {
+  // Element to string
+  const userInput = elem.value;
+
+  // Check for a positive number
+  if (isPositiveNumber(userInput) === false) {
+    window.alert("Please, enter a positive number.");
+    return;
+  }
```

Comment:
This works, but double check what her UI looks like and try and replicate that if you can. you can modify an elements class and add/remove bootstrap classes to them.

Reply:
Agreed. There should be red and green text like the example snippet, so try modifying to add or remove classes like 'text-danger'

Reply:
done!